README.md

```diff
@@ -111,7 +110,11 @@ client.send({
 
 Events can be sent to SignalFx via the `sendEvent` function. The
 event type must be specified, and dimensions and extra event properties
-can be supplied as well.
+can be supplied as well. Also please specify event category: for that get
+option from dictionary `client.EVENT_CATEGORIES`. Different categories of events are supported.
+Available categories of events are `USER_DEFINED`, `ALERT`, `AUDIT`, `JOB`, 
```

Comment:
I think the only one we want people to use here is USER_DEFINED. Could we make this an implied default with a way to override?